docs/cep/query-guide/functions/list/index.md

```diff
@@ -29,15 +29,13 @@ This example shows how to use basic list functions.
 -- Defines `ProductComboStream` having `string` type attributes `product1`, `product2`, and `product3`.
 CREATE STREAM ProductComboStream ( product1 string, product2 string, product3 string);
 
-
 @info(name = 'Create-list')
 -- Create a list with values of `product1`, `product2`, and `product3`.
 insert into NewListStream
```

Comment:
@diana-macrometa - The insert statements have remained with small letters instead of capital like in the other files.

Reply:
Well spotted!